src/scripts/constants.ts

```diff
@@ -20,6 +20,10 @@ export module Constants {
 		export var textAreaInput = "textAreaInput";
 		export var textAreaInputMirror = "textAreaInputMirror";
 
+		// optionsPanel
```

Comment:
Consider commenting this with "popover" or similar, instead of "optionsPanel", as we are likely going to re-use this in future experiences.